app/views/admins/_navbar.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="container">
```

Comment:
later u will have to remove admin navbar and move admin buttons to regular navbar.